docs/pages/includes/desktop-access/desktop-config.mdx

```diff
@@ -17,15 +17,13 @@ windows_desktop_service:
     # Active Directory domain name you are connecting to.
     domain:   '$LDAP_DOMAIN_NAME'
     # LDAP username for authentication. This username must include the domain
-    # NetBIOS name.
+    # NetBIOS name. The use of single quotes here is intentional in order to
+    # avoid the need to escape the backslash (\) character.
     #
     # For example, if your domain is "example.com", the NetBIOS name for it is
-    # likely "EXAMPLE". When connecting as the "Administrator" user, you should
-    # use the format: "EXAMPLE\Administrator".
+    # likely "EXAMPLE". When connecting as the "svc-teleport" user, you should
+    # use the format: "EXAMPLE\svc-teleport".
     username: '$LDAP_USERNAME'
-    # Plain text file containing the LDAP password for authentication.
-    # This is usually the same password you use to login to the Domain Controller.
-    password_file: /var/lib/ldap-pass
```

Comment:
I'm a big fan of removing the `password_file` from a security perspective because clearly plaintext passwords are not secure. I might have missed it, but what error are we returning to users who leave behind this file in their config? Given that this will be the major release where we are bringing desktop access out of preview, I'm not worried about phasing out existing users so I'm onboard with erroring out (as opposed to something less aggressive such as a warning).

Reply:
What do you think about creating a special case for this? Perhaps something more descriptive like "field password_file was deprecated in Teleport 9.0.0 in favor of certificate-based authentication, see {link} for more information. Delete password_file from your teleport.yaml file to fix this error."

We can link to the relevant 9.0.0 release notes where we cover the details.

I think this is an edge case so I'm comfortable with keeping the existing behavior, especially if we are against adding special cases.

Reply:
I'm fine with that, would be a nicer experience for the user. We can do this and schedule it for deletion in Teleport 10.